lib/ScanStream.ts

```diff
@@ -48,7 +48,10 @@ export default class ScanStream extends Readable {
 
     this.opt.redis[this.opt.command](args, (err, res) => {
       if (err) {
-        this.emit("error", err);
+        const error = new Error(
```

Comment:
Should I add something like
```
error.previous = err;
```

or even do something as simple as just modifying the error message (without creating a new Error):
```
err.message = ``${this.opt.command}Stream failed: ${err.message}`
```

Reply:
We should just modify the error message in place since the original `err` may provide useful error stack (given we do provide `showFriendlyErrorStack` option)

Reply:
Thanks, fixed